### PR TITLE
Create BLorient12511

### PR DESCRIPTION
**Pull request by CarstenHoffmannMarburg**
I created a record for BLorient12511. I took three rather short calendaric treatises as guest texts. Strelcyn mentioned them not as contents, but in his remarks. So I assume, he regarded these as guest texts and not as main content.

### CODE REVIEW

File: LondonBritishLibrary/orient/BLorient12511.xml [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+                        <collection>Oriental</collection>
+                        <idno>BL Oriental 12511</idno>
+                        <altIdentifier><idno>Strelcyn 6</idno></altIdentifier>
+                    </msIdentifier>
+                    <msContents>
+                        <msItem xml:id="ms_i1">
+                            <locus from="3r" to="73v"/>
+                            <title ref="LIT1693John"/>
+                            <incipit xml:lang="gez">ብስራተ፡ አብ፡ ትሩፍ፡ ረድእ፡ ተናጋሪ፡ በመለኮተ፡ ዮሐንስ፡ ወልደ፡ ዘብዴዎስ፡ ፍቁረ፡ እግዚእነ፡ ኢየሱስ፡ ክርስቶስ፡ ጸሎቱ፡ ወበረከቱ፡
+                                የሀሉ፡ ምስለ፡ ፍቁሩ፡ <persName ref="PRS14329WaldaMikael">ወልደ፡ ሚካኤል፡</persName> ለዓለመ፡ ዓለም፡ አሜን፡</incipit>
```
Comment (eu-genia) [inline LondonBritishLibrary/orient/BLorient12511.xml:36]: role="owner"

File: LondonBritishLibrary/orient/BLorient12511.xml [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+                                <item xml:id="a6">
+                                    <locus from="1r" to="1v"/>
+                                    <desc type="GuestText">A fragment of <ref type="work" corresp="LIT1398Fethat"/> comparable to 
+                                        <bibl><ptr target="bm:YabetaKrestiyanSalot1944"/><citedRange>10</citedRange></bibl> written in a very 
+                                        poor hand.</desc>
+                                </item> 
+                                <item xml:id="e1">
+                                    <locus target="#100r"/>
+                                    <desc type="OwnershipNote">The second owner 
+                                        <persName ref="PRS14330WaldaTaklaHaym">Walda Takla Hāymānot</persName> and the third owner
```
Comment (eu-genia) [inline LondonBritishLibrary/orient/BLorient12511.xml:104]: role

File: LondonBritishLibrary/orient/BLorient12511.xml [hunk 3]
```diff
@@ -0,0 +1,191 @@
...
+                                    <locus from="1r" to="1v"/>
+                                    <desc type="GuestText">A fragment of <ref type="work" corresp="LIT1398Fethat"/> comparable to 
+                                        <bibl><ptr target="bm:YabetaKrestiyanSalot1944"/><citedRange>10</citedRange></bibl> written in a very 
+                                        poor hand.</desc>
+                                </item> 
+                                <item xml:id="e1">
+                                    <locus target="#100r"/>
+                                    <desc type="OwnershipNote">The second owner 
+                                        <persName ref="PRS14330WaldaTaklaHaym">Walda Takla Hāymānot</persName> and the third owner 
+                                        <persName ref="PRS14331TasfaSeyon">Tasfā Ṣǝyon</persName> mentioned in a note.</desc>
```
Comment (eu-genia) [inline LondonBritishLibrary/orient/BLorient12511.xml:105]: role

**Review by eu-genia — CHANGES_REQUESTED** [review]

**Review by thea-m — APPROVED** [review]
Very nice

**Pull request merged**